doc/user/mgmtd.rst

```diff
@@ -305,6 +305,18 @@ MGMT Configuration commands
     This command dumps the DB specified in the db-name into the file in JSON
     format. This command in not supported for the Operational DB.
 
+.. clicmd:: [no] mgmt copy-config running startup [always]
+
+    This command saves a copy of the running datastore to the startup datastore
+    such that on restart of MGMT daemon (or the entire FRR) the same can be
+    restored and loaded back onto the running datastore.
+
+    If the 'always' option is supplied the contents of the running dtatstore
+    shall be automatically saved onto the startup datastore everytime any
+    configuration change is committed to it. To disable the automatic saving
+    of contents of running datastore to startup datastore the command
+    'no mgmt copy-config running startup always' can be used.
+
```

Comment:
Is this config or exec? I think it's config in which case I don't understand what "always" is for. If this config is present then this new behavior is enabled, otherwise it's not.

I'm not a fan of this functionality. I think when we actually properly support a newer non-implicit config commit that is where we should work on this capability. For implicit config (legacy) this is new behavior that just isn't needed and is confusing to boot.

Reply:
'mgmt copy-config running startup' is exec. And 'mgmt copy-config running startup always' is config. The former is one-time  action just like 'write memory'. Whereas the later is a config switch to turn on auto-saving of running datastore content to startup file.